Misc/NEWS.d/next/Core and Builtins/2022-09-29-15-19-29.gh-issue-94526.wq5m6T.rst

```diff
@@ -0,0 +1,5 @@
+Fix the Python path configuration used to initialized :data:`sys.path` at
+Python startup. ``getpath_basename()`` and ``getpath_dirname()`` functions
```

Comment:
The NEWS entry is meant to be read by a common Python user, not a core dev. `getpath_basename` and `getpath_dirname` are not Python function. Could you rewrite this?